packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -105,19 +120,12 @@ export function buildSmartSnippetQuestionsList(
     );
   };
 
-  const getIsExpanded = (
-    questionAndAnswer: QuestionAnswer,
-    relatedQuestions: QuestionAnsweringRelatedQuestionState[]
-  ) => {
-    const idx = findRelatedQuestionIdx(
-      relatedQuestions,
-      questionAndAnswer.documentId
-    );
-    if (idx === -1) {
-      return false;
-    }
-    return relatedQuestions[idx].expanded;
-  };
+  const getPayloadFromIdentifier = (
```

Comment:
The actions take an object as a parameter instead of a string (`{questionAnswerId: string}`). I felt like this was more descriptive/explicit than just a string type. The controller methods only take a string, since I find it explicit enough with their parameter name and their jsdoc. Plus, I don't think we have a pattern of taking objects with a single value as parameters in controllers.